src/gtimelog/timelog.py

```diff
@@ -1033,7 +1033,7 @@ def parse_correction(self, entry):
         """
         now = None
         date_match = re.match(r'(\d\d):(\d\d)\s+', entry)
-        delta_match = re.match(r'-([1-9]\d?|1\d\d)\s+', entry)
+        delta_match = re.match(r'[\-+]([1-9]\d?|1\d\d)\s+', entry)
```

Comment:
The `-` is not necessary here, `-` is not special when it's the first or the last character inside a `[`...`]` group.

OTOH it doesn't hurt.

Reply:
I assume you mean `\` and I did not know that...
Should it be changed?